python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/management-api-models.j2

```diff
@@ -7,14 +7,16 @@
 {% if management_api_models is arista.avd.defined %}
 !
 management api models
-{%     for provider in management_api_models.providers | arista.avd.natural_sort('name') %}
-{%         if provider.name is arista.avd.defined and provider.name | lower in ['smash', 'sysdb'] %}
+{%     for provider in management_api_models.providers | arista.avd.natural_sort %}
+{%         if provider.name is arista.avd.defined %}
    !
    provider {{ provider.name }}
-{%             for path in provider.paths | arista.avd.natural_sort('path') %}
-{%                 set provider_cli = "path " ~ path.path %}
-{%                 if path.disabled is arista.avd.defined(true) %}
-{%                     set provider_cli = provider_cli ~ " disabled" %}
+{%             for path in provider.paths | arista.avd.natural_sort %}
```

Comment:
The change to sort here will change the behavior which should not happen in a refactor of test vars.

Reply:
Reverted this change to eos j2.